libr/anal/var.c

```diff
@@ -1740,6 +1755,25 @@ R_API void r_anal_var_list_show(RAnal *anal, RAnalFunction *fcn, int kind, int m
 	r_list_free (list);
 }
 
+static bool is_default_argname(const char *name) {
```

Comment:
If i understand this function correctly it can be simplified to just check if the flag starts with “arg” and the very first char after the “g” is a number. No need to loop or check for nullbyte. Simplify this logic because this can be done in 2 lines